src/main/java/picard/vcf/LiftoverVcf.java

```diff
@@ -313,21 +323,155 @@ public static void main(final String[] args) {
         return 0;
     }
 
-    protected static GenotypesContext fixGenotypes(final GenotypesContext originals, final Map<Allele, Allele> reverseComplementAlleleMap) {
+    protected static Locatable getFlippedLocation(final VariantContext source, final LiftOver liftOver) {
+        Interval originalLocus = new Interval(source.getContig(), source.getStart() + 1, source.getEnd() + 1);
```

Comment:
could you please add a comment as to why you're adding +1 to start and end?  I think this method could use docs since it's not obvious what you're doing.